x-pack/plugins/apm/server/lib/helpers/transaction_error_rate.ts

```diff
@@ -48,6 +51,21 @@ export function calculateFailedTransactionRate(
   return failedTransactions / (successfulTransactions + failedTransactions);
 }
 
+export function calculateFailedTransactionRateFromServiceMetrics({
+  failedTransactions,
+  successfulTransactions,
+}: {
+  failedTransactions: number | null;
+  successfulTransactions: number | null;
+}) {
+  if (isNull(failedTransactions) || failedTransactions === 0) {
```

Comment:
This is up to you, just a preference:

```suggestion
  if (!failedTransactions) {
```